wasm-commp/Cargo.toml

```
@@ -0,0 +1,21 @@
+[package]
+authors = ["aidan46"]
+description = "WASM-based CommP calculation"
+edition = "2021"
+name = "wasm-commp"
+version = "0.1.0"
+
+[lib]
+crate-type = ["cdylib"]
+
+[dependencies]
+byte-slice-cast = "1.2.3"
+primitives = { git = "https://github.com/eigerco/polka-storage/", package = "primitives", default-features = false }
```

I'd rather have a commit revision but this also works
```suggestion
primitives = { git = "https://github.com/eigerco/polka-storage/", package = "primitives", branch = "develop", default-features = false }
```